app/views/coronavirus_pages/show.html.erb

```diff
@@ -1,11 +1,11 @@
 <%
   links = [
     {
-      text: 'Coronavirus pages',
+      text: t("coronavirus_pages.show.link_text"),
       href: coronavirus_pages_path
     },
     {
-      text: "#{@coronavirus_page.name} accordions"
+      text: "#{@coronavirus_page.name} #{t("coronavirus_pages.show.accordian_text")}"
```

Comment:
I wonder if it would be more idiomatic for this locale entry to accept the variable rather than being two distinct entities?

Reply:
Thanks! Have updated.